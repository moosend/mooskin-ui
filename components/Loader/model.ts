import { IBoxComponentProps } from '../Box/model';

export interface ILoaderComponentProps extends IBoxComponentProps {
	/** Size of the loader */
	size?: number | string;

	/** width of the spinner */
	spinnerWidth?: string | number;
}
