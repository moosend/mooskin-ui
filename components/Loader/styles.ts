import styled, { keyframes } from 'styled-components';

// Helpers
import { getNumberOrStringValue } from '../Box/styles';

// Models
import { ILoaderComponentProps } from './model';

// Components
import { Box } from '../Box/Box';

// "CSS" Variables
import variables from '../_utils/globals/variables';

export const spin = keyframes`
	0% {
		transform: rotate(0deg);
	}
	100% {
		transform: rotate(360deg);
	}
`;

export const StyledLoader = styled(Box)<ILoaderComponentProps>`
	border: ${(props) => getNumberOrStringValue(props.spinnerWidth)} solid
		${(props) => props.palette?.borderColors.gray1 || variables.borderColors.gray1};
	border-top: ${(props) => getNumberOrStringValue(props.spinnerWidth)} solid
		${(props) => props.palette?.borderColors.primary1 || variables.borderColors.primary1};
	border-radius: 50%;
	width: ${(props) => getNumberOrStringValue(props.size)};
	height: ${(props) => getNumberOrStringValue(props.size)};
	animation: ${spin} 2s linear infinite;
`;

StyledLoader.displayName = 'StyledLoader';
