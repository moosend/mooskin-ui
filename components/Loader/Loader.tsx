import * as React from 'react';

// Mooskin Context HoC that passes context to component props
import { withMooskinContext } from '../Styled/MooskinContextProvider';
import { getNumberOrStringValue } from '../Box/styles';
// Models
import { ILoaderComponentProps } from './model';
// import { getNumberOrStringValue } from '../Box/styles';
// Styled Components
import { StyledLoader } from './styles';

import variables from '../_utils/globals/variables';

/**
 * Loader
 */

export const Loader: React.FC<ILoaderComponentProps> = withMooskinContext(
	({ className = '', size = 50, spinnerWidth = 5, style = {}, ...props }) => {
		return (
			<StyledLoader
				border={`${getNumberOrStringValue(spinnerWidth)} solid ${props.palette?.borderColors.gray1 || variables.borderColors.gray1}`}
				borderTop={`${getNumberOrStringValue(spinnerWidth)} solid	${
					props.palette?.borderColors.primary1 || variables.borderColors.primary1
				}`}
				borderRadius="50%"
				w={getNumberOrStringValue(size)}
				h={getNumberOrStringValue(size)}
				{...props}
			/>
		);
	}
);

Loader.displayName = 'Loader';
