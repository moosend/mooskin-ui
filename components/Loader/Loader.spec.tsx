import * as React from 'react';
import { Loader } from './Loader';

import { mount } from 'enzyme';

describe('Loader', () => {
	test('renders correctly', () => {
		const tree = mount(<Loader />);
		expect(tree).toMatchSnapshot();
	});
});
