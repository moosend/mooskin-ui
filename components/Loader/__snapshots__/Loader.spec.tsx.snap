// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<Loader
  className=""
  size={50}
  spinnerWidth={5}
  style={Object {}}
>
  <Component
    className=""
    palette={
      Object {
        "backgroundColors": Object {
          "accountNavigation": "#FFFFFF",
          "altWhite": "#FFFFFF",
          "autosExpression": "#ffffff",
          "background": "#FAFAFA",
          "black": "#000000",
          "blue400": "#4299E1",
          "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
          "client": "#E0E0E0",
          "createNew": "#FFFFFF",
          "darkgray1": "#777777",
          "darkgray2": "#454545",
          "darkgray3": "#424242",
          "dropdown": "#FFFFFF",
          "editorElement": "#1286FA",
          "editorRepeatable": "#5C50E7",
          "editorRow": "#63B8C8",
          "error": "#FCE2E2",
          "errorIcon": "#DB1D1D",
          "gray1": "#E0E0E0",
          "gray100": "#F0F0F0",
          "gray2": "#E7E7E7",
          "grayAlpha": "rgba(0,0,0,0.17)",
          "grayAlpha100": "rgba(0,0,0,0.06)",
          "grayAlpha400": "rgba(0,0,0,0.36)",
          "green600": "#2F855A",
          "info": "#E8E6FB",
          "infoIcon": "#5548D9",
          "inverseText": "#212121",
          "medgray1": "#757575",
          "medgray2": "#888888",
          "menuDark": "#21262A",
          "menuLight": "#2E3841",
          "orange400": "#ED8936",
          "pink400": "#ED64A6",
          "primary1": "#5548D9",
          "primary2": "#212121",
          "primary3": "#00796B",
          "primary4": "#D32F2F",
          "primary5": "#f0b941",
          "purple400": "#9f7aea",
          "red400": "#f56565",
          "red600": "#C53030",
          "secondary1": "#2E7D32",
          "secondary2": "#757575",
          "secondary3": "#673AB7",
          "secondary4": "#9C27B0",
          "secondary5": "#00796B",
          "secondary6": "#ECF0FC",
          "submenu": "#FFFFFF",
          "success": "#C2F2EA",
          "successIcon": "#007E6A",
          "teal400": "#38B2AC",
          "teal500": "#02999A",
          "text": "#212121",
          "warning": "#FFE9BB",
          "warningIcon": "#B2551F",
          "white": "#FFFFFF",
        },
        "borderColors": Object {
          "accountNavigation": "#FFFFFF",
          "altWhite": "#FFFFFF",
          "autosExpression": "#ffffff",
          "background": "#FAFAFA",
          "black": "#000000",
          "blue400": "#4299E1",
          "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
          "client": "#E0E0E0",
          "createNew": "#FFFFFF",
          "darkgray1": "#777777",
          "darkgray2": "#454545",
          "darkgray3": "#424242",
          "dropdown": "#FFFFFF",
          "editorElement": "#1286FA",
          "editorRepeatable": "#5C50E7",
          "editorRow": "#63B8C8",
          "error": "#FCE2E2",
          "errorIcon": "#DB1D1D",
          "gray1": "#E0E0E0",
          "gray100": "#F0F0F0",
          "gray2": "#E7E7E7",
          "grayAlpha": "rgba(0,0,0,0.17)",
          "grayAlpha100": "rgba(0,0,0,0.06)",
          "grayAlpha400": "rgba(0,0,0,0.36)",
          "green600": "#2F855A",
          "info": "#E8E6FB",
          "infoIcon": "#5548D9",
          "inverseText": "#212121",
          "medgray1": "#757575",
          "medgray2": "#888888",
          "menuDark": "#21262A",
          "menuLight": "#2E3841",
          "orange400": "#ED8936",
          "pink400": "#ED64A6",
          "primary1": "#5548D9",
          "primary2": "#212121",
          "primary3": "#00796B",
          "primary4": "#D32F2F",
          "primary5": "#f0b941",
          "purple400": "#9f7aea",
          "red400": "#f56565",
          "red600": "#C53030",
          "secondary1": "#2E7D32",
          "secondary2": "#757575",
          "secondary3": "#673AB7",
          "secondary4": "#9C27B0",
          "secondary5": "#00796B",
          "secondary6": "#ECF0FC",
          "submenu": "#FFFFFF",
          "success": "#C2F2EA",
          "successIcon": "#007E6A",
          "teal400": "#38B2AC",
          "teal500": "#02999A",
          "text": "#212121",
          "warning": "#FFE9BB",
          "warningIcon": "#B2551F",
          "white": "#FFFFFF",
        },
        "fontColors": Object {
          "accountNavigation": "#FFFFFF",
          "altWhite": "#FFFFFF",
          "autosExpression": "#ffffff",
          "background": "#FAFAFA",
          "black": "#000000",
          "blue400": "#4299E1",
          "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
          "client": "#E0E0E0",
          "createNew": "#212121",
          "darkgray1": "#777777",
          "darkgray2": "#454545",
          "darkgray3": "#424242",
          "dropdown": "#212121",
          "editorElement": "#1286FA",
          "editorRepeatable": "#5C50E7",
          "editorRow": "#63B8C8",
          "error": "#FCE2E2",
          "errorIcon": "#DB1D1D",
          "gray1": "#E0E0E0",
          "gray100": "#F0F0F0",
          "gray2": "#E7E7E7",
          "grayAlpha": "rgba(0,0,0,0.17)",
          "grayAlpha100": "rgba(0,0,0,0.06)",
          "grayAlpha400": "rgba(0,0,0,0.36)",
          "green600": "#2F855A",
          "info": "#E8E6FB",
          "infoIcon": "#5548D9",
          "inverseText": "#212121",
          "medgray1": "#757575",
          "medgray2": "#888888",
          "menuDark": "#21262A",
          "menuLight": "#2E3841",
          "orange400": "#ED8936",
          "pink400": "#ED64A6",
          "primary1": "#5548D9",
          "primary2": "#212121",
          "primary3": "#00796B",
          "primary4": "#D32F2F",
          "primary5": "#f0b941",
          "purple400": "#9f7aea",
          "red400": "#f56565",
          "red600": "#C53030",
          "secondary1": "#2E7D32",
          "secondary2": "#757575",
          "secondary3": "#673AB7",
          "secondary4": "#9C27B0",
          "secondary5": "#00796B",
          "secondary6": "#ECF0FC",
          "submenu": "#FFFFFF",
          "success": "#C2F2EA",
          "successIcon": "#007E6A",
          "teal400": "#38B2AC",
          "teal500": "#02999A",
          "text": "#212121",
          "warning": "#FFE9BB",
          "warningIcon": "#B2551F",
          "white": "#FFFFFF",
        },
        "styles": Object {
          "buttonBorderRadius": "100px",
        },
      }
    }
    size={50}
    spinnerWidth={5}
    style={Object {}}
  >
    <StyledLoader
      className=""
      palette={
        Object {
          "backgroundColors": Object {
            "accountNavigation": "#FFFFFF",
            "altWhite": "#FFFFFF",
            "autosExpression": "#ffffff",
            "background": "#FAFAFA",
            "black": "#000000",
            "blue400": "#4299E1",
            "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
            "client": "#E0E0E0",
            "createNew": "#FFFFFF",
            "darkgray1": "#777777",
            "darkgray2": "#454545",
            "darkgray3": "#424242",
            "dropdown": "#FFFFFF",
            "editorElement": "#1286FA",
            "editorRepeatable": "#5C50E7",
            "editorRow": "#63B8C8",
            "error": "#FCE2E2",
            "errorIcon": "#DB1D1D",
            "gray1": "#E0E0E0",
            "gray100": "#F0F0F0",
            "gray2": "#E7E7E7",
            "grayAlpha": "rgba(0,0,0,0.17)",
            "grayAlpha100": "rgba(0,0,0,0.06)",
            "grayAlpha400": "rgba(0,0,0,0.36)",
            "green600": "#2F855A",
            "info": "#E8E6FB",
            "infoIcon": "#5548D9",
            "inverseText": "#212121",
            "medgray1": "#757575",
            "medgray2": "#888888",
            "menuDark": "#21262A",
            "menuLight": "#2E3841",
            "orange400": "#ED8936",
            "pink400": "#ED64A6",
            "primary1": "#5548D9",
            "primary2": "#212121",
            "primary3": "#00796B",
            "primary4": "#D32F2F",
            "primary5": "#f0b941",
            "purple400": "#9f7aea",
            "red400": "#f56565",
            "red600": "#C53030",
            "secondary1": "#2E7D32",
            "secondary2": "#757575",
            "secondary3": "#673AB7",
            "secondary4": "#9C27B0",
            "secondary5": "#00796B",
            "secondary6": "#ECF0FC",
            "submenu": "#FFFFFF",
            "success": "#C2F2EA",
            "successIcon": "#007E6A",
            "teal400": "#38B2AC",
            "teal500": "#02999A",
            "text": "#212121",
            "warning": "#FFE9BB",
            "warningIcon": "#B2551F",
            "white": "#FFFFFF",
          },
          "borderColors": Object {
            "accountNavigation": "#FFFFFF",
            "altWhite": "#FFFFFF",
            "autosExpression": "#ffffff",
            "background": "#FAFAFA",
            "black": "#000000",
            "blue400": "#4299E1",
            "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
            "client": "#E0E0E0",
            "createNew": "#FFFFFF",
            "darkgray1": "#777777",
            "darkgray2": "#454545",
            "darkgray3": "#424242",
            "dropdown": "#FFFFFF",
            "editorElement": "#1286FA",
            "editorRepeatable": "#5C50E7",
            "editorRow": "#63B8C8",
            "error": "#FCE2E2",
            "errorIcon": "#DB1D1D",
            "gray1": "#E0E0E0",
            "gray100": "#F0F0F0",
            "gray2": "#E7E7E7",
            "grayAlpha": "rgba(0,0,0,0.17)",
            "grayAlpha100": "rgba(0,0,0,0.06)",
            "grayAlpha400": "rgba(0,0,0,0.36)",
            "green600": "#2F855A",
            "info": "#E8E6FB",
            "infoIcon": "#5548D9",
            "inverseText": "#212121",
            "medgray1": "#757575",
            "medgray2": "#888888",
            "menuDark": "#21262A",
            "menuLight": "#2E3841",
            "orange400": "#ED8936",
            "pink400": "#ED64A6",
            "primary1": "#5548D9",
            "primary2": "#212121",
            "primary3": "#00796B",
            "primary4": "#D32F2F",
            "primary5": "#f0b941",
            "purple400": "#9f7aea",
            "red400": "#f56565",
            "red600": "#C53030",
            "secondary1": "#2E7D32",
            "secondary2": "#757575",
            "secondary3": "#673AB7",
            "secondary4": "#9C27B0",
            "secondary5": "#00796B",
            "secondary6": "#ECF0FC",
            "submenu": "#FFFFFF",
            "success": "#C2F2EA",
            "successIcon": "#007E6A",
            "teal400": "#38B2AC",
            "teal500": "#02999A",
            "text": "#212121",
            "warning": "#FFE9BB",
            "warningIcon": "#B2551F",
            "white": "#FFFFFF",
          },
          "fontColors": Object {
            "accountNavigation": "#FFFFFF",
            "altWhite": "#FFFFFF",
            "autosExpression": "#ffffff",
            "background": "#FAFAFA",
            "black": "#000000",
            "blue400": "#4299E1",
            "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
            "client": "#E0E0E0",
            "createNew": "#212121",
            "darkgray1": "#777777",
            "darkgray2": "#454545",
            "darkgray3": "#424242",
            "dropdown": "#212121",
            "editorElement": "#1286FA",
            "editorRepeatable": "#5C50E7",
            "editorRow": "#63B8C8",
            "error": "#FCE2E2",
            "errorIcon": "#DB1D1D",
            "gray1": "#E0E0E0",
            "gray100": "#F0F0F0",
            "gray2": "#E7E7E7",
            "grayAlpha": "rgba(0,0,0,0.17)",
            "grayAlpha100": "rgba(0,0,0,0.06)",
            "grayAlpha400": "rgba(0,0,0,0.36)",
            "green600": "#2F855A",
            "info": "#E8E6FB",
            "infoIcon": "#5548D9",
            "inverseText": "#212121",
            "medgray1": "#757575",
            "medgray2": "#888888",
            "menuDark": "#21262A",
            "menuLight": "#2E3841",
            "orange400": "#ED8936",
            "pink400": "#ED64A6",
            "primary1": "#5548D9",
            "primary2": "#212121",
            "primary3": "#00796B",
            "primary4": "#D32F2F",
            "primary5": "#f0b941",
            "purple400": "#9f7aea",
            "red400": "#f56565",
            "red600": "#C53030",
            "secondary1": "#2E7D32",
            "secondary2": "#757575",
            "secondary3": "#673AB7",
            "secondary4": "#9C27B0",
            "secondary5": "#00796B",
            "secondary6": "#ECF0FC",
            "submenu": "#FFFFFF",
            "success": "#C2F2EA",
            "successIcon": "#007E6A",
            "teal400": "#38B2AC",
            "teal500": "#02999A",
            "text": "#212121",
            "warning": "#FFE9BB",
            "warningIcon": "#B2551F",
            "white": "#FFFFFF",
          },
          "styles": Object {
            "buttonBorderRadius": "100px",
          },
        }
      }
      size={50}
      spinnerWidth={5}
      style={Object {}}
    >
      <Box
        className="sc-gsDKAQ cAMSac"
        palette={
          Object {
            "backgroundColors": Object {
              "accountNavigation": "#FFFFFF",
              "altWhite": "#FFFFFF",
              "autosExpression": "#ffffff",
              "background": "#FAFAFA",
              "black": "#000000",
              "blue400": "#4299E1",
              "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
              "client": "#E0E0E0",
              "createNew": "#FFFFFF",
              "darkgray1": "#777777",
              "darkgray2": "#454545",
              "darkgray3": "#424242",
              "dropdown": "#FFFFFF",
              "editorElement": "#1286FA",
              "editorRepeatable": "#5C50E7",
              "editorRow": "#63B8C8",
              "error": "#FCE2E2",
              "errorIcon": "#DB1D1D",
              "gray1": "#E0E0E0",
              "gray100": "#F0F0F0",
              "gray2": "#E7E7E7",
              "grayAlpha": "rgba(0,0,0,0.17)",
              "grayAlpha100": "rgba(0,0,0,0.06)",
              "grayAlpha400": "rgba(0,0,0,0.36)",
              "green600": "#2F855A",
              "info": "#E8E6FB",
              "infoIcon": "#5548D9",
              "inverseText": "#212121",
              "medgray1": "#757575",
              "medgray2": "#888888",
              "menuDark": "#21262A",
              "menuLight": "#2E3841",
              "orange400": "#ED8936",
              "pink400": "#ED64A6",
              "primary1": "#5548D9",
              "primary2": "#212121",
              "primary3": "#00796B",
              "primary4": "#D32F2F",
              "primary5": "#f0b941",
              "purple400": "#9f7aea",
              "red400": "#f56565",
              "red600": "#C53030",
              "secondary1": "#2E7D32",
              "secondary2": "#757575",
              "secondary3": "#673AB7",
              "secondary4": "#9C27B0",
              "secondary5": "#00796B",
              "secondary6": "#ECF0FC",
              "submenu": "#FFFFFF",
              "success": "#C2F2EA",
              "successIcon": "#007E6A",
              "teal400": "#38B2AC",
              "teal500": "#02999A",
              "text": "#212121",
              "warning": "#FFE9BB",
              "warningIcon": "#B2551F",
              "white": "#FFFFFF",
            },
            "borderColors": Object {
              "accountNavigation": "#FFFFFF",
              "altWhite": "#FFFFFF",
              "autosExpression": "#ffffff",
              "background": "#FAFAFA",
              "black": "#000000",
              "blue400": "#4299E1",
              "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
              "client": "#E0E0E0",
              "createNew": "#FFFFFF",
              "darkgray1": "#777777",
              "darkgray2": "#454545",
              "darkgray3": "#424242",
              "dropdown": "#FFFFFF",
              "editorElement": "#1286FA",
              "editorRepeatable": "#5C50E7",
              "editorRow": "#63B8C8",
              "error": "#FCE2E2",
              "errorIcon": "#DB1D1D",
              "gray1": "#E0E0E0",
              "gray100": "#F0F0F0",
              "gray2": "#E7E7E7",
              "grayAlpha": "rgba(0,0,0,0.17)",
              "grayAlpha100": "rgba(0,0,0,0.06)",
              "grayAlpha400": "rgba(0,0,0,0.36)",
              "green600": "#2F855A",
              "info": "#E8E6FB",
              "infoIcon": "#5548D9",
              "inverseText": "#212121",
              "medgray1": "#757575",
              "medgray2": "#888888",
              "menuDark": "#21262A",
              "menuLight": "#2E3841",
              "orange400": "#ED8936",
              "pink400": "#ED64A6",
              "primary1": "#5548D9",
              "primary2": "#212121",
              "primary3": "#00796B",
              "primary4": "#D32F2F",
              "primary5": "#f0b941",
              "purple400": "#9f7aea",
              "red400": "#f56565",
              "red600": "#C53030",
              "secondary1": "#2E7D32",
              "secondary2": "#757575",
              "secondary3": "#673AB7",
              "secondary4": "#9C27B0",
              "secondary5": "#00796B",
              "secondary6": "#ECF0FC",
              "submenu": "#FFFFFF",
              "success": "#C2F2EA",
              "successIcon": "#007E6A",
              "teal400": "#38B2AC",
              "teal500": "#02999A",
              "text": "#212121",
              "warning": "#FFE9BB",
              "warningIcon": "#B2551F",
              "white": "#FFFFFF",
            },
            "fontColors": Object {
              "accountNavigation": "#FFFFFF",
              "altWhite": "#FFFFFF",
              "autosExpression": "#ffffff",
              "background": "#FAFAFA",
              "black": "#000000",
              "blue400": "#4299E1",
              "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
              "client": "#E0E0E0",
              "createNew": "#212121",
              "darkgray1": "#777777",
              "darkgray2": "#454545",
              "darkgray3": "#424242",
              "dropdown": "#212121",
              "editorElement": "#1286FA",
              "editorRepeatable": "#5C50E7",
              "editorRow": "#63B8C8",
              "error": "#FCE2E2",
              "errorIcon": "#DB1D1D",
              "gray1": "#E0E0E0",
              "gray100": "#F0F0F0",
              "gray2": "#E7E7E7",
              "grayAlpha": "rgba(0,0,0,0.17)",
              "grayAlpha100": "rgba(0,0,0,0.06)",
              "grayAlpha400": "rgba(0,0,0,0.36)",
              "green600": "#2F855A",
              "info": "#E8E6FB",
              "infoIcon": "#5548D9",
              "inverseText": "#212121",
              "medgray1": "#757575",
              "medgray2": "#888888",
              "menuDark": "#21262A",
              "menuLight": "#2E3841",
              "orange400": "#ED8936",
              "pink400": "#ED64A6",
              "primary1": "#5548D9",
              "primary2": "#212121",
              "primary3": "#00796B",
              "primary4": "#D32F2F",
              "primary5": "#f0b941",
              "purple400": "#9f7aea",
              "red400": "#f56565",
              "red600": "#C53030",
              "secondary1": "#2E7D32",
              "secondary2": "#757575",
              "secondary3": "#673AB7",
              "secondary4": "#9C27B0",
              "secondary5": "#00796B",
              "secondary6": "#ECF0FC",
              "submenu": "#FFFFFF",
              "success": "#C2F2EA",
              "successIcon": "#007E6A",
              "teal400": "#38B2AC",
              "teal500": "#02999A",
              "text": "#212121",
              "warning": "#FFE9BB",
              "warningIcon": "#B2551F",
              "white": "#FFFFFF",
            },
            "styles": Object {
              "buttonBorderRadius": "100px",
            },
          }
        }
        size={50}
        spinnerWidth={5}
        style={Object {}}
        transition="0.2s all"
      >
        <Component
          className="sc-gsDKAQ cAMSac"
          palette={
            Object {
              "backgroundColors": Object {
                "accountNavigation": "#FFFFFF",
                "altWhite": "#FFFFFF",
                "autosExpression": "#ffffff",
                "background": "#FAFAFA",
                "black": "#000000",
                "blue400": "#4299E1",
                "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                "client": "#E0E0E0",
                "createNew": "#FFFFFF",
                "darkgray1": "#777777",
                "darkgray2": "#454545",
                "darkgray3": "#424242",
                "dropdown": "#FFFFFF",
                "editorElement": "#1286FA",
                "editorRepeatable": "#5C50E7",
                "editorRow": "#63B8C8",
                "error": "#FCE2E2",
                "errorIcon": "#DB1D1D",
                "gray1": "#E0E0E0",
                "gray100": "#F0F0F0",
                "gray2": "#E7E7E7",
                "grayAlpha": "rgba(0,0,0,0.17)",
                "grayAlpha100": "rgba(0,0,0,0.06)",
                "grayAlpha400": "rgba(0,0,0,0.36)",
                "green600": "#2F855A",
                "info": "#E8E6FB",
                "infoIcon": "#5548D9",
                "inverseText": "#212121",
                "medgray1": "#757575",
                "medgray2": "#888888",
                "menuDark": "#21262A",
                "menuLight": "#2E3841",
                "orange400": "#ED8936",
                "pink400": "#ED64A6",
                "primary1": "#5548D9",
                "primary2": "#212121",
                "primary3": "#00796B",
                "primary4": "#D32F2F",
                "primary5": "#f0b941",
                "purple400": "#9f7aea",
                "red400": "#f56565",
                "red600": "#C53030",
                "secondary1": "#2E7D32",
                "secondary2": "#757575",
                "secondary3": "#673AB7",
                "secondary4": "#9C27B0",
                "secondary5": "#00796B",
                "secondary6": "#ECF0FC",
                "submenu": "#FFFFFF",
                "success": "#C2F2EA",
                "successIcon": "#007E6A",
                "teal400": "#38B2AC",
                "teal500": "#02999A",
                "text": "#212121",
                "warning": "#FFE9BB",
                "warningIcon": "#B2551F",
                "white": "#FFFFFF",
              },
              "borderColors": Object {
                "accountNavigation": "#FFFFFF",
                "altWhite": "#FFFFFF",
                "autosExpression": "#ffffff",
                "background": "#FAFAFA",
                "black": "#000000",
                "blue400": "#4299E1",
                "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                "client": "#E0E0E0",
                "createNew": "#FFFFFF",
                "darkgray1": "#777777",
                "darkgray2": "#454545",
                "darkgray3": "#424242",
                "dropdown": "#FFFFFF",
                "editorElement": "#1286FA",
                "editorRepeatable": "#5C50E7",
                "editorRow": "#63B8C8",
                "error": "#FCE2E2",
                "errorIcon": "#DB1D1D",
                "gray1": "#E0E0E0",
                "gray100": "#F0F0F0",
                "gray2": "#E7E7E7",
                "grayAlpha": "rgba(0,0,0,0.17)",
                "grayAlpha100": "rgba(0,0,0,0.06)",
                "grayAlpha400": "rgba(0,0,0,0.36)",
                "green600": "#2F855A",
                "info": "#E8E6FB",
                "infoIcon": "#5548D9",
                "inverseText": "#212121",
                "medgray1": "#757575",
                "medgray2": "#888888",
                "menuDark": "#21262A",
                "menuLight": "#2E3841",
                "orange400": "#ED8936",
                "pink400": "#ED64A6",
                "primary1": "#5548D9",
                "primary2": "#212121",
                "primary3": "#00796B",
                "primary4": "#D32F2F",
                "primary5": "#f0b941",
                "purple400": "#9f7aea",
                "red400": "#f56565",
                "red600": "#C53030",
                "secondary1": "#2E7D32",
                "secondary2": "#757575",
                "secondary3": "#673AB7",
                "secondary4": "#9C27B0",
                "secondary5": "#00796B",
                "secondary6": "#ECF0FC",
                "submenu": "#FFFFFF",
                "success": "#C2F2EA",
                "successIcon": "#007E6A",
                "teal400": "#38B2AC",
                "teal500": "#02999A",
                "text": "#212121",
                "warning": "#FFE9BB",
                "warningIcon": "#B2551F",
                "white": "#FFFFFF",
              },
              "fontColors": Object {
                "accountNavigation": "#FFFFFF",
                "altWhite": "#FFFFFF",
                "autosExpression": "#ffffff",
                "background": "#FAFAFA",
                "black": "#000000",
                "blue400": "#4299E1",
                "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                "client": "#E0E0E0",
                "createNew": "#212121",
                "darkgray1": "#777777",
                "darkgray2": "#454545",
                "darkgray3": "#424242",
                "dropdown": "#212121",
                "editorElement": "#1286FA",
                "editorRepeatable": "#5C50E7",
                "editorRow": "#63B8C8",
                "error": "#FCE2E2",
                "errorIcon": "#DB1D1D",
                "gray1": "#E0E0E0",
                "gray100": "#F0F0F0",
                "gray2": "#E7E7E7",
                "grayAlpha": "rgba(0,0,0,0.17)",
                "grayAlpha100": "rgba(0,0,0,0.06)",
                "grayAlpha400": "rgba(0,0,0,0.36)",
                "green600": "#2F855A",
                "info": "#E8E6FB",
                "infoIcon": "#5548D9",
                "inverseText": "#212121",
                "medgray1": "#757575",
                "medgray2": "#888888",
                "menuDark": "#21262A",
                "menuLight": "#2E3841",
                "orange400": "#ED8936",
                "pink400": "#ED64A6",
                "primary1": "#5548D9",
                "primary2": "#212121",
                "primary3": "#00796B",
                "primary4": "#D32F2F",
                "primary5": "#f0b941",
                "purple400": "#9f7aea",
                "red400": "#f56565",
                "red600": "#C53030",
                "secondary1": "#2E7D32",
                "secondary2": "#757575",
                "secondary3": "#673AB7",
                "secondary4": "#9C27B0",
                "secondary5": "#00796B",
                "secondary6": "#ECF0FC",
                "submenu": "#FFFFFF",
                "success": "#C2F2EA",
                "successIcon": "#007E6A",
                "teal400": "#38B2AC",
                "teal500": "#02999A",
                "text": "#212121",
                "warning": "#FFE9BB",
                "warningIcon": "#B2551F",
                "white": "#FFFFFF",
              },
              "styles": Object {
                "buttonBorderRadius": "100px",
              },
            }
          }
          size={50}
          spinnerWidth={5}
          style={Object {}}
          transition="0.2s all"
        >
          <div
            className="sc-bdvvtL buKeIX sc-gsDKAQ cAMSac"
            size={50}
            style={Object {}}
          />
        </Component>
      </Box>
    </StyledLoader>
  </Component>
</Loader>
`;
